dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/main/java/org/apache/dolphinscheduler/plugin/alert/http/HttpSender.java

```diff
@@ -112,7 +112,8 @@ private void createHttpRequest(String msg) throws MalformedURLException, URISynt
             //GET request add param in url
             setMsgInUrl(msg);
             URL unencodeUrl = new URL(url);
-            URI uri = new URI(unencodeUrl.getProtocol(), unencodeUrl.getHost(), unencodeUrl.getPath(), unencodeUrl.getQuery(), null);
+            URI uri = new URI(unencodeUrl.getProtocol(), unencodeUrl.getPort() == -1 ? unencodeUrl.getHost()
```

Comment:
Can we directly use `new URL(url).toURI()` ?

Reply:
yes， of cource